packages/app-store/routing-forms/components/react-awesome-query-builder/widgets.tsx

```diff
@@ -371,6 +371,7 @@ const widgets = {
   ButtonGroup,
   Conjs,
   Provider,
+  EmailWidget: EmailField,
```

Comment:
Instead of import EmailField directly in config we add it too to widgets like others.